assets/js/pluginsidebar/components/sidebar/index.js

```diff
@@ -60,9 +61,16 @@ class Sidebar extends React.PureComponent {
       shareUrl: PropTypes.string,
       revision: PropTypes.string,
     }).isRequired,
+    appleNewsNotices: PropTypes.arrayOf(PropTypes.shape({
+      dismissed: PropTypes.bool,
+      dismissible: PropTypes.bool,
+      message: PropTypes.string,
+      type: PropTypes.string,
+    })),
```

Comment:
If this isn't required, then it needs to be added to default props with a value of `[]`.